src/bodypart.h

```diff
@@ -130,17 +130,19 @@ struct body_part_type {
         int bionic_slots_ = 0;
 };
 
+constexpr int number_of_parts = 12;
 class body_part_set
 {
     private:
-        std::bitset<num_bp> parts;
 
-        explicit body_part_set( const std::bitset<num_bp> &other ) : parts( other ) { }
+        std::bitset<number_of_parts> parts;
```

Comment:
So long as this is being implemented via `bp->token` you might as well leave this as `std::bitset<num_bp>`.

Once you want to migrate `body_part_set` away from being based on the enum I suggest replacing it with a `cata::flat_set<bodypart_id>` or `cata::flat_set<bodypart_str_id>`, or you could do some kind of bitset based on the `int_id` variant, but we don't currently have a dynamic bitset class, as far as I know, so that would be tricky.